tools/ds-identify

```diff
@@ -1258,7 +1258,7 @@ dscheck_OpenStack() {
 
     # LP: #1715241 : arch other than intel are not identified properly.
     case "$DI_UNAME_MACHINE" in
-        i?86|x86_64) :;;
```

Comment:
what about the one up here?

Reply:
Sorry, I didn't notice that before. I'll check again.